word_guess.rb

```diff
@@ -0,0 +1,269 @@
+require 'colorize'
+
+class WordGuess
+
+  # creates a game object that contains all the game methods
+  # sets the word
+  def initialize
+    @guess = "" # initializes w/ empty guess
+    @wrong_count = 5 # Maybe will change this difficulty level
```

Comment:
This is a bit confusing based on the variable name. It makes me think that this is the number of guesses that the player has gotten wrong. Consider re-naming or clarifying throughout to make it a bit more clear of the purpose.